### DISCUSSION TITLE
Chat room font size should be able to change

### DISCUSSION THREAD

**Discussion by sekatol** [ppy/osu#30065] [Ideas] [answered]
Now, I've found the only way to enlarge the font size in chat room is to change the UI scaling. But I haven't found a way to enlarge the font size only in chat room. Is there any way to do this?

**Comment by peppy** [comment]
If you're finding it too small, we probably need to increase it slightly.

Can you take a screenshot for reference?

**Reply by sekatol** [reply]
- Without UI scaling:

![withoutScaling](https://github.com/user-attachments/assets/59933f8a-05f0-4912-9642-6e5e608a70b5)

This problem is more serious in CJK chat channels, because ideographs is more complex to read using small fonts as follows.

![cjkWithoutScaling](https://github.com/user-attachments/assets/9fa5cbea-a5dd-4c16-b8c1-2e32c29c40a4)

- With UI scaling:

![withScaling](https://github.com/user-attachments/assets/7539238f-a544-4d24-8b87-f569cd45cd06)

But the results screen gets too big:

https://github.com/user-attachments/assets/e0fc86f4-b0af-44dc-9ebb-4d88317f66a5

Related: #30045

**Comment by vatei** [comment]
Tbh I feel like the chat font is too small everywhere

**Reply by bdach** [reply]
If that's the feeling you have then maybe the UI scale slider in settings is the solution for you.

**Reply by vatei** [reply]
I did scale the UI to 1.10x before, but reverted it back to 1x because some UI elements didn't fit properly (especially results screen). 
And the chat still felt small in comparison unfortunately

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30095

**Comment by jai-x** [comment]
Sorry to post against an answered discussion, but the original design of the chat overlay did have buttons for adjusting the font size;
https://www.figma.com/design/f8b2dHp9LJCMOqYP4mdrPZ/Client%2FChat?node-id=0-1&node-type=canvas&t=XHX4nTd7qopjAI5R-0

![Screenshot 2024-10-04 at 11 34 21](https://github.com/user-attachments/assets/0e0a0286-9f79-4942-91c2-e74b02af30a7)


I never got round to implementing this. Unsure if this is still a desired feature.

**Reply by peppy** [reply]
Dunno. I feel like the size should match with all other game sizes and not require a local adjustment. Any adjustments should be game wide (whether via UI Scale or otherwise).
